projects/Open-Rival-test/src/tests/TestWalkerComponent.cpp

```diff
@@ -13,7 +13,8 @@ namespace TestWalkerComponent {
 
 SCENARIO("WalkerComponent can plan a route", "[components][movement-component]")
 {
-    World world(5, 5, false);
+    World world(6, 6, false);
+    World largeWorld(1001, 1001, false);
```

Comment:
You can move this inside the relevant `GIVEN` block to prevent it being reinstantiated for every test.

Reply:
Will do!